release-checks.sh

```diff
@@ -57,6 +60,16 @@ then
     exit 1
 fi
 
+if [[ $(uname) == "Darwin" ]]; then
+    DOCKER_MEM=$(docker info | grep Memory | awk '{print ($3 + 0)}')
+    HAS_ENOUGH_MEM=$(echo "$DOCKER_MEM > $MIN_DOCKER_MEM" | bc)
+    if [[ ${HAS_ENOUGH_MEM} -eq 0 ]]; then
+        echo "Error: Docker is not configured with enough memory."
+        echo "Please increase memory available to Docker to at least ${MIN_DOCKER_MEM} gb"
```

Comment:
Can we change this to the correct unit symbol, which is `GB` (upper-case)?

Reply:
https://github.com/elastic/rally/pull/914/commits/601eb0ebb08c31769662843d1cd5b3b9adde2c9f